docs/src/main/asciidoc/vault-transit.adoc

```diff
@@ -0,0 +1,267 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+= Quarkus - Working with HashiCorp Vault's Transit Secret Engine
+
+include::./attributes.adoc[]
+:root-token: s.5VUS8pte13RqekCB2fmMT3u2
+:client-token: s.s93BVzJPzBiIGuYJHBTkG8Uw
+:config-file: application.properties
+:extension-status: preview
+:base-guide: link:vault[Vault guide]
```

Comment:
I fixed the URL here, otherwise the generated html still points to the .adoc.

Reply:
then, this is probably wrong: https://github.com/quarkusio/quarkus/blame/master/docs/src/main/asciidoc/spring-security.adoc#L144 (that is where I copied the approach from)
and this: https://github.com/quarkusio/quarkus/blame/master/docs/src/main/asciidoc/index.adoc#L59-L60
correct?

Reply:
should the vault guides be referenced into the index.adoc?

Reply:
Yes, it's incorrect, I'll go fix it, thanks for pointing it out.

As for `index.adoc`, we don't really maintain that one anymore.

Reply:
(and it's not used on the website at all)